UnitTest/ApiCompatTests.cs

```diff
@@ -79,9 +79,7 @@ public void TestGeolocation()
         {
             Assert.True(InvokeGeolocation().Wait(1000));
         }
-#endif
 
-#if ENABLE_WORKSTATION_TESTS 

```

Comment:
The first #if is unmatched in this PR. Intentional?

Reply:
the #ifs are matched - the PR doesn't show it well